app/celery/process_delivery_status_result_tasks.py

```diff
@@ -199,22 +212,6 @@ def _get_notification_parameters(notification_platform_status: dict) -> Tuple[st
     return payload, reference, notification_status, number_of_message_parts, price_in_millicents_usd
 
 
-def _calculate_pricing(price_in_millicents_usd: float, notification: Notification, notification_status: str,
-                       number_of_message_parts: int):
-
-    """ Calculate pricing """
-    current_app.logger.info("Calculate Pricing")
-    if price_in_millicents_usd > 0.0:
-        notification.status = notification_status
-        notification.segments_count = number_of_message_parts
-        notification.cost_in_millicents = price_in_millicents_usd
-        dao_update_notification(notification)
-    else:
-        # notification_id -  is the UID in the database for the notification
-        # status - is the notification platform status generated earlier
-        update_notification_status_by_id(notification_id=notification.id, status=notification_status)
-
-
```

Comment:
Why are we removing this as a hidden method? I get that it's only used in this file, but I don't get what that has to do with the ticket itself.